rocks-bin/src/upload.rs

```diff
@@ -0,0 +1,19 @@
+use eyre::eyre;
+use std::env;
+
+use eyre::Result;
+use rocks_lib::{config::Config, project::Project, upload::upload_from_project};
+
+pub async fn upload(config: Config) -> Result<()> {
+    let project = Project::current()?.unwrap();
+
+    upload_from_project(
+        &project,
+        env::var("ROCKS_API_KEY")
+            .map_err(|_| eyre!("No API key provided! Please set the $ROCKS_API_KEY variable."))?,
```

Comment:
suggestion(for-another-issue): let's add a config option that allows us to use `libsecret` to fetch API keys,